src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -872,6 +876,26 @@ export default class DefaultAudioVideoController
     return true;
   }
 
+  updateLocalVideoFromPolicy(): boolean {
+    if (this.mayNeedRenegotiationForSimulcastLayerChange) {
+      this.logger.info('Needs regenotiation for local video simulcast layer change');
+      this.mayNeedRenegotiationForSimulcastLayerChange = false;
+      return false;
+    }
+
+    // Update bandwidth without renegotiation
+    this.logger.info('Updating local video from policy without renegotiation');
+    if (this.meetingSessionContext.enableSimulcast) {
+      // `AttachMediaInputTask` will update sender's simulcast streams encoding parameters of the local video transceiver
+      new AttachMediaInputTask(this.meetingSessionContext).run();
```

Comment:
Don't we also have some logic for simulcast in ReceiveVideoInputTask? 
https://github.com/aws/amazon-chime-sdk-js/blob/master/src/task/ReceiveVideoInputTask.ts#L50

Reply:
I don't think that logic matters, since the reason its integrated into the `VideoStreamIndex` is so we can advertise the new max bitrates in the subscribe message.  Since we intentionally avoid the subscribe it doesn't really matter (downlinks should be using avg bitrate).